__mocks__/data/mailchimp/response.json

```diff
@@ -0,0 +1,17 @@
+{
+    "https://us20.api.mailchimp.com/3.0/lists/df42a82d07/members/0b63fa319d113aede8b7b409e4fc6437": {
+        "type": "https://mailchimp.com/developer/marketing/docs/errors/",
+        "title": "Forbidden",
```

Comment:
If you have `status` in your mock Response you should use that  (Be it 200/ 403 ) in your response instead of hardcoding it